tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/ProcessBasicCheck.cs

```diff
@@ -163,17 +163,18 @@ public static bool Run(ProcessInfo process, IRegistryService? registryService =
             }
             else if (!ok)
             {
-                AnsiConsole.WriteLine(TracingWithInstaller);
```

Comment:
Should we add something in the `else` case to cover macos etc too?

Reply:
dd-dotnet is not available on macos

Reply:
Hmmm... But `dd-trace` _is_... is that going to be a problem? 🤔

Reply:
I don't think this is going to be a problem, but we should at least remove the check command on MacOS (currently it will complain that it can't find dd-dotnet)